### PR TITLE
Add wireless switch

### PR DESCRIPTION
**Pull request by JELoohuis**
-**Added support for wireless switches**
  Added support for 'wxkg' Tuya devices.
  The following capabilities were implemented:
  - battery_percentage (measure_battery)

  Additionally the following capabilities were implemented only in flows:
  - knob_switch_mode_1
  - switch_modeX, where X can be any value used to denote the sub-switch

### CODE REVIEW

File: drivers/button/driver.flow.compose.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
+{
+  "actions": [
```
Comment (bobvandevijver) [inline drivers/button/driver.flow.compose.json:2]: Lets remove the actions

**Review by bobvandevijver — CHANGES_REQUESTED** [review]

File: drivers/button/driver.compose.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "$extends": "tuya",
+  "class": "button",
+  "name": {
+    "en": "Wireless Button",
+    "nl": "Draadloze Knop"
+  },
+  "capabilities": []
```
Comment (bobvandevijver) [inline drivers/button/driver.compose.json:8]:
```suggestion
  "capabilities": ["measure_battery"]
```

**Comment by deejayreissue** [comment]
Amazing, any ideas when there will be a test version to check out with the new device type please?

**Comment by bobvandevijver** [comment]
@deejayreissue You can subscribe to the releases to receive a notification when a new test release is made. We hope to create a release as soon as possible, but we are currently waiting for a couple of blocking assets.

**Pull request merged**